docs/modules/trino/pages/usage-guide/connect_to_trino.adoc

```diff
@@ -1,52 +1,57 @@
-= Connect to Trino
+= Connecting to Trino
+:trino-jdbc: https://trino.io/docs/current/client/jdbc.html
+:starburst-odbc: https://docs.starburst.io/data-consumer/clients/odbc.html
+:trino-cli: https://trino.io/docs/current/client/cli.html
+:dbeaver: https://dbeaver.io
+:trino-python-client: https://github.com/trinodb/trino-python-client
 
 Trino offers an SQL endpoint via a HTTP API.
-An https://trino.io/docs/current/client/jdbc.html[open source JDBC driver] does exist as well as a proprietary https://docs.starburst.io/data-consumer/clients/odbc.html[ODBC driver].
+An {trino-jdbc}[open source JDBC driver] does exist as well as a proprietary {starburst-odbc}[ODBC driver].
 
 Multiple SQL clients can be used to connect to Trino as described below.
 
 == Prerequisites
 The Trino server must be reachable from your local computer.
 It is typically secured via HTTPS with a certificated provided by a xref:secret-operator:secretclass.adoc[SecretClass].
 
-As most setups use a self-signed certificate, the easiest way is to disable the Trino cert validation as the following guide will do.
-A production-ready solution would be to extract the `ca.crt` from the SecretClass and provide it to the Trino client.
+As most setups use a self-signed certificate, that cannot be verified without the CA certificate, the easiest way is to disable the Trino cert validation as the following guide will do, however this is not secure!
+To use certificate validation instead, you need to extract the `ca.crt` file from the the SecretClass and provide it to the Trino client.
```

Comment:
Technically it is the Secret created/referenced by the SecretClass - and only in case autotls is used. But I guess this should be sufficient for now

Reply:
ok yeah i'd leave it the way it is, someone deailng with TLS will hopefully know what is meant here